appservice/src/tree/DeploymentsTreeItem.ts

```diff
@@ -0,0 +1,80 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SiteConfig } from 'azure-arm-website/lib/models';
+import * as path from 'path';
+import { AzureParentTreeItem, DialogResponses, GenericTreeItem, IActionContext } from 'vscode-azureextensionui';
+import KuduClient from 'vscode-azurekudu';
+import { DeployResult } from 'vscode-azurekudu/lib/models';
+import { editScmType } from '../editScmType';
+import { ext } from '../extensionVariables';
+import { getKuduClient } from '../getKuduClient';
+import { ScmType } from '../ScmType';
+import { DeploymentTreeItem } from './DeploymentTreeItem';
+import { ISiteTreeRoot } from './ISiteTreeRoot';
+
+export class DeploymentsTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
+    public static contextValueConnected: string = 'deploymentsConnected';
+    public static contextValueUnconnected: string = 'deploymentsUnconnected';
+    public contextValue: string = DeploymentsTreeItem.contextValueUnconnected;
```

Comment:
nit: Don't set a default value for contextValue if you're just going to overwrite it in the constructor